prediction_market_agent/agents/microchain_agent/code_functions.py

```diff
@@ -1,48 +1,44 @@
-import json
-import tempfile
-
-from autogen.coding import CodeBlock, LocalCommandLineCodeExecutor
+# from autogen.coding import CodeBlock, LocalCommandLineCodeExecutor
```

Comment:
Needs to be reverted once autogen stuff is fixed.

Reply:
Please merge this branch (https://github.com/gnosis/prediction-market-agent/commits/peter/nftdeployments-with-autogen) in, it fixes autogen